serving-llama2-70b-on-l4-gpus/text-generation-interface.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: llama-2-70b
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: llama-2-70b
+  template:
+    metadata:
+      labels:
+        app: llama-2-70b
+    spec:
+      containers:
+      - name: llama-2-70b
+        image: ghcr.io/huggingface/text-generation-inference:1.0.3
+        resources:
+          limits:
+            nvidia.com/gpu: 2
+        env:
+        - name: MODEL_ID
+          value: meta-llama/Llama-2-70b-chat-hf
+        - name: NUM_SHARD
```

Comment:
It might be helpful to give a brief bullet point on what some of these less self-explanatory environment variables do, such as `NUM_SHARD`, so people can adapt this to other inference infra setups. For example, if you try googling `text-generation-interface \"NUM_SHARD\"` I don't see any relevant docs about this env var (maybe I'm just not finding them), and so the user will be forced to read the [comments](https://github.com/huggingface/text-generation-inference/blob/v1.0.3/launcher/src/main.rs#L119-L124) in the main.rs file for answers, which is not ideal.

Reply:
I updated with some basic explanation.